arcgis-ios-sdk-samples/Scenes/Animate 3D graphic/README.metadata.json

```diff
@@ -1,19 +1,48 @@
 {
     "category": "Scenes",
-    "description": "This sample demonstrates how to animate a graphic's position and rotation and have the camera follow it.",
+    "description": "An `AGSOrbitGeoElementCameraController` follows a graphic while the graphic's position and rotation are animated.",
```

Comment:
I forget if platform specific API should be included in README title section and metadata description field. Will double-check.

Reply:
This seems to be acceptable. I'll check on the preview site to see if they are correct in the page `<head>`.

```sh
find . -type f -name "*.json" -print0 | xargs -0 grep -n '"description".*`'
```

It gives me 3 results: `Maps/Change basemap`, `/Maps/Map loaded` and `/Scenes/Animate 3D graphic`.